appveyor.yml

```diff
@@ -36,7 +37,7 @@ cache:
   # different branch is built.
   - "%BRANCH_CACHE_DIR%"        # e.g. C:\cache\feature_abc123
   - "%CACHE_DIR%\\master"
-  - "%CACHE_DIR%\\cardano-sl-0.4"
+  - "%CACHE_DIR%\\cardano-sl-%VERSION%"
```

Comment:
Nice one!

Reply:
@deepfire https://www.appveyor.com/docs/environment-variables/

if it's not possible to merge the two variables, they should have corresponding comments to update both

Reply:
@domenkozar, done: https://ci.appveyor.com/project/jagajaga/cardano-sl/build/1.0.14818#L7